base_tier_validation_forward/models/tier_review.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2020 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class TierReview(models.Model):
+    _inherit = "tier.review"
+    _order = "sequence"
+
+    name = fields.Char(
+        compute="_compute_definition_data", store=True, related=False, translate=True
```

Comment:
IMO we need a comment here mentioning why we keep `translate=True` :wink:

Reply:
Added `NOTE` in field name. Thank you for review.